rust_src/remacs-sys/lib.rs

```diff
@@ -94,7 +94,7 @@ pub enum Lisp_Type {
     Lisp_Float = 7,
 }
 
-#[repr(isize)]
```

Comment:
Based on discussions in other threads, I believe that changing this to repr(C) is the right call.